### ISSUE TITLE
cpu inference : Warning: max_batch_requests=8

### ISSUE THREAD

**Issue by michelgirault** [huggingface/text-embeddings-inference#454]
### System Info

when i start the server with tei cpu
i get the warning: 
 {"timestamp":"2024-12-13T13:35:00.921111Z","level":"WARN","message":"**Backend does not support a batch size > 8**","target":"text_embeddings_router","filename":"router/src/lib.rs","line_number":258}

 {"timestamp":"2024-12-13T13:35:00.921158Z","level":"WARN","message":"forcing `max_batch_requests=8`","target":"text_embeddings_router","filename":"router/src/lib.rs","line_number":259}

currents settings:
 {
"timestamp":"2024-12-13T13:34:54.360800Z",
"level":"INFO",
"message":"Args { model_id: \"BAA*/***-****-**-v1.5\", 
revision: None, 
tokenization_workers: None,
 dtype: None, 
pooling: Some(Cls), 
max_concurrent_requests: 512, 
max_batch_tokens: 65536, 
max_batch_requests: None, 
max_client_batch_size: 512, 
auto_truncate: false, 
default_prompt_name: None, 
default_prompt: None, 
hf_api_token: None, 
hostname: \"0.0.0.0\", 
port: 4000, 
uds_path: \"/tmp/text-embeddings-inference-server\",
 huggingface_hub_cache: Some(\"/data\"), 
payload_limit: 2000000, 
api_key: None, 
json_output: true, 
otlp_endpoint: None, 
otlp_service_name: \"text-embeddings-inference.server\", 
cors_allow_origin: None 
}

I was wondering if this normal and how to fix that

### Information

- [X] Docker
- [ ] The CLI directly

### Tasks

- [ ] An officially supported command
- [ ] My own modifications

### Reproduction

1. select the docker images : ghcr.io/huggingface/text-embeddings-inference:cpu-latest-grpc
2. run
3. access log 

### Expected behavior

logs extracts:

Backend does not support a batch size > 8
forcing `max_batch_requests=8

**Comment by OlivierDehaene** [comment]
Yes it is normal. On CPU we limit the batch size as the workload cannot be parallelized as much as with GPUs.
